README.md

```diff
@@ -310,7 +310,7 @@ provides a number of options to do this. First of all, it doesn't eliminate the
 of doing this in fully manual mode using means provided by the Gear Protocol - whether
 you want to use a set of the `msg::send` functions from the `gstd` crate for interacting
 from one application with another, or you want to use functionality provided by the `gclient`
-crate for interacting from off-chain code with an on-chain application, or [TDB about JS],
+crate for interacting from off-chain code with an on-chain application, or you can create your JavaScript application using [sails-js](js/README.md),
```

Comment:
Here we need to say how to interact with a contract written in Sails in manual mode, i.e. w/o generated client.